### PR TITLE
Add first msk topic for cbc

### PR DESCRIPTION
**Pull request by martskins**

### CODE REVIEW

**Review by MarcinGinszt — APPROVED** [review]

File: CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,4 @@
 *                        @utilitywarehouse/pubsub
 /*/customer-billing/     @utilitywarehouse/org-billing-experience
 /*/finance/              @utilitywarehouse/finance-platform
+/*/cbc/                  @utilitywarehouse/org-cashback
```
Comment (catalin-ilea) [inline CODEOWNERS:4]: please remove as per https://github.com/utilitywarehouse/kafka-cluster-config/tree/main/prod-aws/kafka-shared-msk#contributing

File: dev-aws/kafka-shared-msk/cbc/cbc.tf [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  name = "cbc.FraudEvents"
+
+  replication_factor = 3
+  partitions         = 15
+
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
+    "retention.ms"          = "-1"
+    "local.retention.ms"    = "3600000" # keep data locally for 1 hour
```
Comment (LukeG07) [inline dev-aws/kafka-shared-msk/cbc/cbc.tf:11]: I'd probably increase local retention to like a day to account for any consumer lag

**Review by LukeG07 — CHANGES_REQUESTED** [review]

File: dev-aws/kafka-shared-msk/cbc/cbc.tf (hunk 2 above)
Comment (LukeG07) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:11]: Although it's dev so it doesn't matter I guess, feel free to resolve

**Review by LukeG07 — APPROVED** [review]

File: CODEOWNERS (hunk 1 above)
Comment (martskins) [reply CODEOWNERS]: Oh sorry, missed that! Removed!

File: dev-aws/kafka-shared-msk/cbc/cbc.tf (hunk 2 above)
Comment (martskins) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:11]: fair point and maybe we should clarify with the help of the pubsub folks before doing this in prod. As you say, in dev it's probably pointless to do more than this!

File: dev-aws/kafka-shared-msk/cbc/cbc.tf [hunk 3]
```diff
@@ -0,0 +1,16 @@
+resource "kafka_topic" "FraudEvents" {
+  name = "cbc.FraudEvents"
+
+  replication_factor = 3
+  partitions         = 15
+
+  config = {
+    "remote.storage.enable" = "true"
+    "retention.bytes"       = "-1"
+    "retention.ms"          = "-1"
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/cbc/cbc.tf:10]: Do you need infinite retention for dev?

Comment (martskins) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:10]: I'm a bit concerned that not having it be infinite could cause issues (not with existing services of course). If it is a problem for MSK we can analyse changing this value, but to be fair, we don't even have that many events in dev to begin with.

Comment (sbuliarca) [reply dev-aws/kafka-shared-msk/cbc/cbc.tf:10]: >If it is a problem for MSK

No issues for MSK, just wanted to raise it to see if it was intentional.

**Pull request merged**